packages/plugin-react/src/index.js

```diff
@@ -1,7 +1,12 @@
-module.exports = {
-  name: 'react',
-  init: (client, React = window.React) => {
+module.exports = class BugsnagReactPlugin {
+  constructor (React = window.React) {
     if (!React) throw new Error('cannot find React')
```

Comment:
Should we be throwing here if we take the idea that the only configuration that should blow up is an invalid API KEY to its full extent?

Reply:
If we don't blow up here, when a user attempts to do `const ErrorBoundary = getPlugin('react')`, that will return `undefined`, so it will unexpectedly blow up when they try to render. The former seems more desirable.